[docs] Document Nutanix machineset sample (#1679)
openshift/windows-machine-config-operator: docs/machineset-nutanix.md

# Creating a Nutanix Windows MachineSet

> _\<windows_server_image_name\>_ should be replaced with the full name of an image of a [supported version](wmco-prerequisites.md#supported-windows-server-versions) of Windows, pre-uploaded to the Prism-Central/Prism-Element where the Machine VMs will be created with.

Reviewer: ```suggestion
Replace _\<windows_server_image_name\>_ with the full name of an image of a [supported version](wmco-prerequisites.md#supported-windows-server-versions) of Microsoft Windows that is pre-uploaded to the Prism-Central/Prism-Element where the Machine VMs will be created.
```

> _\<windows_server_image_name\>_ should be replaced with the full name of an image of a [supported version](wmco-prerequisites.md#supported-windows-server-versions) of Windows, pre-uploaded to the Prism-Central/Prism-Element where the Machine VMs will be created with.

Reviewer: I'm assuming that "Windows" on line 3 refers to a Microsoft Windows OS and not the Windows Machine Config Operator. To avoid any potential confusion, consider replacing "of Windows" with "of the Windows operating system".

Author: Changed according to the suggestion.

For example, `nutanix-windows-server-2022`

> _\<infrastructure_id\>_ should be replaced with the output of:

Reviewer: Replace _\<infrastructure_id\>_ with the output:

> _\<infrastructure_id\>_ should be replaced with the output of:

Reviewer: ```suggestion
Replace _\<infrastructure_id\>_ with the output:
```

> Replace _\<infrastructure_id\>_ should be replaced with the output:

Reviewer: ```suggestion
Replace _\<infrastructure_id\>_ with the output:
```

Author: Fixed.
